codegen/manifest.go

```diff
@@ -25,8 +33,9 @@ type AppManifestKindPermission struct {
 }
 
 type SimpleManifest struct {
-	Props    AppManifestProperties
-	AllKinds []Kind
+	Props                AppManifestProperties
+	AllKinds             []Kind
+	ManifestCustomRoutes []AppManifestCustomRoute
```

Comment:
I think for consistency this should probably be added to the `AppManifestProperties` type rather than to the interface/implementation.